packages/web/pages/_app.tsx

```diff
@@ -63,6 +65,7 @@ const DEFAULT_LANGUAGE = "en";
 
 function MyApp({ Component, pageProps }: AppProps) {
   useAmplitudeAnalytics({ init: true });
+  initSuperflow("GbkZQ3DHV4rsGongQlYg", { projectId: "2059891376305922" });
```

Comment:
NIT: IDK how big superflow is, but we could look into dynamic imports for only if we're on stage to reduce bundle size

Reply:
@MaxMillington https://www.npmjs.com/package/@usesuperflow/client

It's not too big